pkg/datastore/mysql/mysql.go

```diff
@@ -129,3 +281,21 @@ func buildDataSourceName(url, database, usernameFile, passwordFile string) (stri
 
 	return fmt.Sprintf("%s:%s@tcp(%s)/%s", username, password, url, database), nil
 }
+
+func buildGetQuery(table string) string {
+	// TODO: make kinds from datastore package public
+	if table == "Project" {
+		// TODO: Should we find this using WHERE condition on `extra` field or on `data->>"$.id"`` attribute (need to create index on that attribute to reduce recomputing cost).
```

Comment:
Maybe instead of "extra" field, we can add a "ProjectId" field to this table as a stable field like other tables.

Reply:
Thanks for your comment 🙏 Since the data size of this table is not that big, we would use `data->>"$.id"` directly not the `extra` field for this case 🙆‍♀️

Reply:
👍